lightning/src/events/mod.rs

```diff
@@ -1173,6 +1190,12 @@ impl Writeable for Event {
 					(0, payment_id, required),
 				})
 			},
+			&Event::ConnectionNeeded { ref node_announcement } => {
+				35u8.write(writer)?;
+				write_tlv_fields!(writer, {
+					(0, node_announcement, required),
+				})
+			},
```

Comment:
I guess we shouldn't need to serialize these since we don't save buffered messages across restarts.